src/ecrc-frontend/src/components/page/orgverification/OrgVerification.js

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+
+import "../page.css";
+import Header from "../../base/header/Header";
+import Footer from "../../base/footer/Footer";
+import SideCard from "../../base/sideCard/SideCard";
+import { Button } from "../../base/button/Button";
+import Table from "../../composite/table/Table";
+
+export default function OrgVerification({
+  page: {
+    pageLayout: { header, sideCard1, sideCard2 },
```

Comment:
can we name the `sidecard` here something more meaningful so it describes the type of sidecard we are passing into this component?

Reply:
Yeah, Im redoing the whole way we are currently using sidecards so we can just call them by name, as many or few as we want so we arnt stuck to a pagelayout format